doc/user/core.md

```diff
@@ -66,41 +66,25 @@ avoid copies.
 
 ---
 
- * `MakeAlias(a, mat, rows, cols, strict=true)`
+ * `MakeAlias(a, mat, rows, cols, offset=0, strict=true)`
```

Comment:
Can you document the version for vectors too?

Reply:
done